noxfile.py

```diff
@@ -5,28 +5,51 @@
 
 import nox
 
-nox.options.reuse_existing_virtualenvs = True
+nox.options.reuse_existing_virtualenvs = False
 
+# Sphinx output and source directories
+BUILD_DIR = "_build"
```

Comment:
I added variables that I will use in a future PR to clean up the other documentation build steps. I just wanted to be able to test things locally so that is the main change here.